seacrowd/sea_datasets/xnli/xnli.py

```diff
@@ -0,0 +1,218 @@
+# Some code referenced from https://huggingface.co/datasets/xnli/blob/main/xnli.py
+
+"""
+The Cross-lingual Natural Language Inference (XNLI) corpus is a crowd-sourced collection of 5,000 test and 2,500 dev pairs for the MultiNLI corpus. The pairs are annotated with textual entailment and translated into 14 languages: French, Spanish, German, Greek, Bulgarian, Russian, Turkish, Arabic, Vietnamese, Thai, Chinese, Hindi, Swahili and Urdu. This results in 112.5k annotated pairs. Each premise can be associated with the corresponding hypothesis in the 15 languages, summing up to more than 1.5M combinations.
+"""
+
+import csv
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@InProceedings{conneau2018xnli,
+  author = "Conneau, Alexis
+        and Rinott, Ruty
+        and Lample, Guillaume
+        and Williams, Adina
+        and Bowman, Samuel R.
+        and Schwenk, Holger
+        and Stoyanov, Veselin",
+  title = "XNLI: Evaluating Cross-lingual Sentence Representations",
+  booktitle = "Proceedings of the 2018 Conference on Empirical Methods
+               in Natural Language Processing",
+  year = "2018",
+  publisher = "Association for Computational Linguistics",
+  location = "Brussels, Belgium",
+}
+"""
+
+_DATASETNAME = "xnli"
+
+_DESCRIPTION = """\
+XNLI is a subset of a few thousand examples from MNLI which has been translated into a 14 different languages (some low-ish resource). As with MNLI, the goal is to predict textual entailment (does sentence A imply/contradict/neither sentence B) and is a classification task (given two sentences, predict one of three labels).
+"""
+
+_HOMEPAGE = "https://github.com/facebookresearch/XNLI"
+
+# We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+_LANGUAGES = ["tha", "vie"]
+_LANGUAGE_MAPPER = {"tha": "th", "vie": "vi"}
+
+_LICENSE = Licenses.CC_BY_NC_4_0.value
+
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: {
+        "train": "https://dl.fbaipublicfiles.com/XNLI/XNLI-MT-1.0.zip",
+        "test": "https://dl.fbaipublicfiles.com/XNLI/XNLI-1.0.zip",
+    }
+}
+
+_SUPPORTED_TASKS = [Tasks.TEXTUAL_ENTAILMENT]
+
+_SOURCE_VERSION = "1.1.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class XNLI(datasets.GeneratorBasedBuilder):
```

Comment:
add `Dataset` on Dataloader Class Name
```suggestion
class XNLIDataset(datasets.GeneratorBasedBuilder):
```